File: twoSum.py
def twoSum(nums, target):
    total = []
    for i in range(len(nums)):
        for j in range(i + 1, len(nums)):
            if nums[i] + nums[j] == target:
                total.append(nums[i])
                total.append(nums[j])
                return total  # return immediately when found
    return []  # return empty if no pair found


print(twoSum([1, 2, 3, 4, 5], 9))  # Output: [4, 5]

File: Javascript/Array/twosum.js
var twoSum = function(nums, target) {
    let total = [];
    for (let i = 0; i < nums.length; i++) {
        for (let j = i + 1; j < nums.length; j++) {
            if (nums[i] + nums[j] === target) {
                total.push(nums[i], nums[j]);
                return total; // return immediately when found
            }
        }
    }
    return []; // return empty if no pair found
};

console.log(twoSum([1, 2, 3, 4, 5], 9)); // Output: [3, 4]
